src/state/history/reducer.js

```diff
@@ -0,0 +1,25 @@
+import { createReducer } from 'redux-yo';
+import moment from 'moment-timezone';
+import { historyActions } from './actions';
+
+const initialState = {};
+
+export const historyReducer = createReducer(
+  {
+    [historyActions.setHistory]: (state, mobiHistory = [], xlmHistory = []) => {
+      const formatData = ({ Data }) => Data.sort((a, b) => a.time - b.time).map((el, idx) => {
+        const elCopy = el;
+        const d = moment.unix(el.time);
+        elCopy.labelDate = `${d.month() + 1}/${d.date() + 1}`;
```

Comment:
I think it better to save Date object in the store, but formatting on the view side with `moment.format` function.